Makefile

```diff
@@ -1,8 +1,8 @@
 pull-scripts:
 	./scripts/pull-scripts
 
-release:
-	./scripts/release-assets
+stage-release:
+	./scripts/stage-release
```

Comment:
If I understand correctly, release is no longer needed because release is not a straightforward copy and PR?

Reply:
Yup. We always regenerate the released/ directory based on the current contents of dev-v2.5 and release-v2.5 on merges to a source branch that feeds into it

Reply:
The caveat here is that, if you made changes directly to dev-v2.5, they would not be updated until either 1) you make the next merge into dev-v2.5-source or 2) you make the merge for the changes into dev-v2.5 and then make a separate PR to regenerate the released/ directory only by running `make stage-release`.

Reply:
The reason for this is that the make sync command run on the release-v2.5 branch is based off of the upstream git repository for dev-v2.5, not based on the current contents in your dev-v2.5 branch. 

We don't support local sync / validation workflows in the scripts

Reply:
That reminds me, we'll need to update release workflow.